src/blip/BLIPConnection.cc

```diff
@@ -626,7 +626,7 @@ namespace litecore { namespace blip {
 
     void Connection::setWebSocket(WebSocket *webSocket, const fleeceapi::AllocedDict &options) {
         _compressionLevel = kDefaultCompressionLevel;
-        auto levelP = options[kCompressionLevelOption];
+        auto levelP = options.get(kCompressionLevelOption);
```

Comment:
Why doesn't `operator[]` work here?

Reply:
gcc complains about ambiguity of the [] operator, I don't know why it isn't able to take the one with parameter const char*.
The same happens in https://github.com/couchbase/couchbase-lite-core/pull/494 .